docs/howto/openstack/nova/move-instance-between-places.md

```diff
@@ -0,0 +1,119 @@
+# Move an instance from one region to another
```

Comment:
In #62 (which has landed in the interim), we changed the terminology to say "server" when we previously said "instance".

Also, we use the gerund rather than the indicative in section headings.

So this should say:

```markdown
# Moving a server from one region to another
```